.cci.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+// Documentation: https://github.com/coreos/coreos-ci/blob/master/README-upstream-ci.md
+
+cosaPod(buildroot: true) {
+    checkout scm
+
+    fcosBuild(make: true)
+
+    stage("Build metal+live") {
+        shwrap("cd /srv/fcos && cosa build metal")
+        shwrap("cd /srv/fcos && cosa buildextend-live")
+    }
+    stage("Test ISO") {
+        shwrap("cd /srv/fcos && kola testiso --cosa-build builds/latest/x86_64/meta.json -S")
```

Comment:
Hmm, I think we should be able to just `cosa kola testiso -S` here to let cosa pass the `--cosa-build` flag automatically.

Reply:
Yeah you're right, though I am hoping soon to teach `kola` directly to do the right thing.

Let's get this in since it's green and working and do other cleanups/enhancements as followups?

Reply:
Yup, didn't mean it to be blocking.